### ISSUE TITLE
[D&D 5E] PHB Classes incomplete

### ISSUE THREAD

**Issue by Rauifreid** [BahamutDragon/pcgen#864]
In the core Players handbook of the 5e data, all the classes have no walking speed or stat adjustments

**Comment by Rauifreid** [comment]
Furthermore all the subclasses seem to be missing

**Comment by BahamutDragon** [comment]
Mate,

For the race, when you select your race and subrace (if that is an option), does a walking speed appear on the character sheet? 
Likewise for the stat adjustments appear?

I am not aware of anything described as a subclass in the PH? Do you mean the class archetypes that typically become available for selection at 4th level?

**Comment by Rauifreid** [comment]
I am sorry, it seems that my issues came more from me not understanding how the program worked than there being anything wrong with the data. I have looked more at it and it looks like the walking speeds as they are implemented are working correctly. And I used subclass incorrectly, meaning subrace, but I expected them to be within the classes themselves. I have now found that they are in another place. I am sorry for wasting your time.
